apps/src/blockly/addons/cdoGenerator.ts

```diff
@@ -13,10 +13,13 @@ export default function initializeGenerator(
   (
     blocklyWrapper.getGenerator() as ExtendedJavascriptGenerator
   ).translateVarName = function (name: string) {
-    return (Blockly.JavaScript.nameDB_ as GoogleBlockly.Names).getName(
+    let varName = (Blockly.JavaScript.nameDB_ as GoogleBlockly.Names).getName(
       name,
       Blockly.VARIABLE_CATEGORY_NAME
     );
+    // Play Lab (aka Studio) variables are contained within the Globals namespace.
+    if (Blockly.varsInGlobals) varName += 'Globals.';
```

Comment:
You put in your comment that the generated code is `'pointsGlobals. = pointsGlobals. + 1;\n'` (which matches this line). Should it instead be `Globals.points = ....`?

Reply:
Wow, I have no idea how that happened! This wouldn't even work like this, but I definitely had it working before break. Great catch. I've updated the logic, re-tested locally, and updated the PR description.  The generated code is now `Globals.points = Globals.points + 1;` for this example.